packages/web/styles/globals.css

```diff
@@ -5,6 +5,22 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer base {
+  ::-webkit-scrollbar {
+    width: 8px;
```

Comment:
Actually I think 8 it a bit too small to find for cursor. Let's do 10px.

Also- do you know if there's any way to not apply this style on mac?

Reply:
> Actually I think 8 it a bit too small to find for cursor. Let's do 10px.

👍 

> Also- do you know if there's any way to not apply this style on mac?

There's no native CSS way, so we'd have to create a script that detects the browser and add a class to the body depending on the platform